veros/setup/global_4deg/global_four_degree.py

```diff
@@ -21,10 +21,13 @@ class GlobalFourDegreeSetup(VerosSetup):
     This setup demonstrates:
      - setting up a realistic model
      - reading input data from external files
+     - including Indonesian throughflow
      - implementing surface forcings
      - applying a simple ice mask
 
     `Adapted from pyOM2 <https://wiki.cen.uni-hamburg.de/ifm/TO/pyOM2/4x4%20global%20model>`_.
+
+    Courtesy of Franka Jesse, Utrecht University
```

Comment:
Maybe put some changelog in here or so? It's not like she wrote the whole thing.

Reply:
Good idea!